prereise/gather/flexibilitydata/human_in_the_loop/demo/functions.py

```diff
@@ -0,0 +1,336 @@
+# from datetime import datetime
+from datetime import timedelta
+
+# import os
+# import sys
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+
+# from sklearn.preprocessing import MinMaxScaler, StandardScaler
+from sklearn.preprocessing import StandardScaler
+from tslearn.clustering import TimeSeriesKMeans
+from tslearn.utils import to_time_series_dataset
+
+# import torch
+# import torch.nn as nn

```

Comment:
Idk, they weren't commented out before. I did that because the linter complains. I will keep them there just in case Sienna might want to revisit this in the repo sometime in the future.